database/src/pool/postgres.rs

```diff
@@ -1891,7 +1894,14 @@ where
                     benchmark_request
                 SET
                     status = $1,
-                    completed_at = NOW()
+                    completed_at = NOW(),
+                    duration_ms = (
```

Comment:
What happens if there is no job in the job queue attached to the request? You can test e.g. by adding a `WHERE` clause that is always false. Will the query return an error?